src/pages/inquiry/list/index.tsx

```diff
@@ -150,6 +150,19 @@ const InquiryList: React.FC = () => {
 						<p>{row.message}</p>
 					</div>
 				</div>
+
+				<h5>{translation["table-row-information"]["common-information"].reply}</h5>
+				{row.replies && row.replies.length ? (
+					row.replies.map((reply: any, index: number) => (
```

Comment:
we can give the type as a string instead of any. right?

Reply:
Yes